### PR TITLE
Fix for Issue #449 Qt errors on startup.

### PR DESCRIPTION
**Pull request by Ramuh**
windowHandle() is not initialized until the Window is shown. Moved the slot connect to main.cpp file after show(). Had to change the OnFramebufferSizeChanged(); to a slot.

### CODE REVIEW

File: src/citra_qt/bootmanager.h [hunk 1]
```diff
@@ -128,12 +128,12 @@ class GRenderWindow : public QWidget, public EmuWindow
...
 
     void OnClientAreaResized(unsigned width, unsigned height);
-
-    void OnFramebufferSizeChanged();
-
+	
 public slots:
     void moveContext();  // overridden
 
+	void OnFramebufferSizeChanged();
```
Comment (archshift) [inline src/citra_qt/bootmanager.h]: Spaces, not tabs

File: src/citra_qt/main.cpp [hunk 2]
```diff
@@ -160,6 +160,10 @@ GMainWindow::GMainWindow()
 
     show();
 
+	#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
```
Comment (archshift) [inline src/citra_qt/main.cpp]: Spaces, not tabs

**Comment by Ramuh** [comment]
Fixed the tabs to spaces.

**Comment by neobrain** [comment]
This is somewhat ugly. Does putting that connect() into an overridden QWidget::showEvent work? (might want to call QWidget::showEvent() explicitly in the overridden function to make things work properly)

**Comment by Ramuh** [comment]
Does work.

File: src/citra_qt/main.cpp [hunk 3]
```diff
@@ -159,7 +159,7 @@ GMainWindow::GMainWindow()
     setWindowTitle(window_title.c_str());
 
     show();
-
+
```
Comment (neobrain) [inline src/citra_qt/main.cpp]: trailing whitespace

File: src/citra_qt/bootmanager.h [hunk 4]
```diff
@@ -145,4 +145,7 @@ public slots:
 
     /// Device id of keyboard for use with KeyMap
     int keyboard_id;
+
+protected:
+    void showEvent(QShowEvent * event) override;
```
Comment (neobrain) [inline src/citra_qt/bootmanager.h]: no space before `*`

File: src/citra_qt/bootmanager.cpp [hunk 5]
```diff
@@ -335,3 +331,11 @@ void GRenderWindow::OnClientAreaResized(unsigned width, unsigned height)
 void GRenderWindow::OnMinimalClientAreaChangeRequest(const std::pair<unsigned,unsigned>& minimal_size) {
     setMinimumSize(minimal_size.first, minimal_size.second);
 }
+
+void GRenderWindow::showEvent(QShowEvent * event) {
+    QWidget::showEvent(event);
+
+    #if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
```
Comment (neobrain) [inline src/citra_qt/bootmanager.cpp]: The reason for doing this here deserves a comment ;)

File: src/citra_qt/bootmanager.cpp [hunk 6]
```diff
@@ -335,3 +331,11 @@ void GRenderWindow::OnClientAreaResized(unsigned width, unsigned height)
...
+        connect(this->windowHandle(), SIGNAL(screenChanged(QScreen*)), this, SLOT(OnFramebufferSizeChanged()));
+    #endif
+}
```
Comment (neobrain) [inline src/citra_qt/bootmanager.cpp]: no newline at end of file

**Comment by neobrain** [comment]
@Ramuh Is there anything holding up addressing the outstanding PR comments?

**Comment by neobrain** [comment]
Author is not responding.

**Pull request closed without merging**